### PR TITLE
#590: Minimal SLC with conda support

### LINKED ISSUE #590: Build minimal SLC language container with conda support
# Background 

- See https://github.com/exasol/script-languages-release/issues/503

# Acceptance Criteria

- Adapt the UDFClient to use dependencies not located in the default paths
- Build a minimal SLC container with Conda support

### PR DESCRIPTION
**Pull request by tkilias**
Fixes #590

### All Submissions:

* [ ] Check if your pull request goes to the correct bash branch (develop or master)?
* [ ] Is the title of the Pull Request correct?
* [ ] Is the title of the corresponding issue correct?
* [ ] Have you updated the changelog?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../../pulls) for the same update/change?  
* [ ] Are you mentioning the issue which this PullRequest fixes ("Fixes...")

### When integrating to Develop branch:

1. [ ] Remember to merge with "Squash and Merge"

### When integrating to Main branch:

1. [ ] Remember to merge with "Merge"

### CODE REVIEW

File: flavors/README.md [hunk 1]
```diff
@@ -72,6 +72,17 @@ The Python-3.8-minimal flavor provides the language Python 3.8 with only absolut
   - Available languages: Python 3.8
   - [Here](https://github.com/exasol/script-languages/tree/master/flavors/python-3.8-minimal-EXASOL-6.2.0/FLAVOR_DESCRIPTION.md) you can find more details about the flavor, such as the provided packages.
 
+### template-Exasol-all-python-3.8-conda
+
+The template-Exasol-all-python-3.8-conda flavor provides the language Python 3.8  with only absolute necessary packages. This allows very simple customization because you control almost all dependencies. In contrast to Python-3.8-minimal, it also allows the installation of conda packages.
+
+- template-Exasol-all-python-3.8-conda
+  - Base Image: ubuntu:20.04
+  - Compatible Exasol Versions: 7.1.\*, 7.0.\*
```
Comment (tomuben) [inline flavors/README.md:81]:
```suggestion
  - Compatible Exasol Versions: 8.0.0, 7.1.\*, 7.0.\*
```

File: flavors/README.md [hunk 2]
```diff
@@ -72,6 +72,17 @@ The Python-3.8-minimal flavor provides the language Python 3.8 with only absolut
...
+  - Available languages: Python 3.8
+  - Supported Package Manager: conda, pip
```
Comment (tomuben) [inline flavors/README.md:83]: For sake of completeness, should we add apt to the list?

**Comment by tkilias** [comment]
Merge before https://github.com/exasol/script-languages-release/pull/667, if necessary with failing CI for the package diff,

File: flavors/README.md (hunk 1 above)
Comment (tkilias) [reply flavors/README.md]: done

File: flavors/README.md (hunk 2 above)
Comment (tkilias) [reply flavors/README.md:83]: no, because it is not in the user accessible part, it is only in the build deps, test deps and conda deps.

**Review by tomuben — APPROVED** [review]

**Pull request merged**